resources/linux/rpm/code.spec.template

```diff
@@ -11,6 +11,8 @@ Icon:     @@NAME@@.xpm
 Requires: @@DEPENDENCIES@@
 AutoReq:  0
 
+%global __provides_exclude ^lib.*$
```

Comment:
Why not make this specific to currently exposed libraries which are from the application data directory ?

```
%global __provides_exclude_from ^(%{_datadir}/@@NAME@@/.*\\.so.*)$
```